app/src/main/admin/components/Touchstones/List/TouchstoneListPage.tsx

```diff
@@ -20,6 +21,7 @@ export class TouchstoneListPageComponent extends React.Component<PageProperties<
         return <div>
             <AdminPageHeader />
             <PageArticle title={TouchstoneListPageComponent.title}>
+                <CreateTouchstoneForm/>
```

Comment:
Should remove this before merging, as it's not wired up.

Reply:
Could you instead use a flag (e.g. in development settings, but not in production) so hide it?